adam-rms.code-workspace

```diff
@@ -5,7 +5,7 @@
 		}
 	],
 	"settings": {
-		"editor.tabSize": 4,
+		"editor.tabSize": 2,
```

Comment:
https://google.github.io/styleguide/jsguide.html#formatting-indent

Is the above link conflicting with eslint? (https://github.com/google/gts/issues/626)

Reply:
yep, although i thought i'd only changed this on my local config.

Reply:
@cherry-john shall we revert this then?

Reply:
makes most sense, will do in current changes